docs/output.md

```diff
@@ -0,0 +1,144 @@
+# Output
+
+This document describes `semgrep` output and the information provided after
+running the program. By default output is sent to `stdout` in a user-readable
+format. JSON output may also be specified with the `--json` flag.
+
+Contents:
+
+* [Default](#default)
+* [JSON](#json)
+
+## Default
+
+By default, `semgrep` outputs results to `stdout`. The output looks like:
+
+```
+<finding-file-path>
+rule:<rule-id>: <rule-message>
+<finding-line-number>: <finding-line-code>
```

Comment:
Is this actually helpful to document? I'm on the fence.

Reply:
Yup, I agree! I don't have strong feelings either way.

Reply:
I'll leave it in for now - we can always iterate on the doc itself and the broader documentation discussion.